test/cli.test.ts

```
@@ -0,0 +1,229 @@
+
+test('single file input - no colors', async () => {
+  const tempFile = await createTempFile('Test content');
+  
+  try {
+    const result = await runCLI([tempFile, '--no-colors']);
+    
+    expect(result.exitCode).toBe(0);
+    expect(result.stdout).toMatch(/^test\.txt \(/);
+    // Should not contain ANSI color codes
+    // eslint-disable-next-line no-control-regex
+    expect(result.stdout).not.toMatch(/\x1b\[[0-9;]*m/);
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)